tox.ini

```diff
@@ -0,0 +1,8 @@
+[pycodestyle]
+ignore = E111,E114,E731
+max-line-length=80
+
+[flake8]
+ignore = E111,E114,E731
+max-line-length=80
+exclude = .git,.idea,__pycache__,scripts/cpplint.py
```

Comment:
Our linter isn't itself lint free? :)

Reply:
Not even close. It doesn't pass `gpylint` either.